includes/manage.form.inc

```diff
@@ -64,44 +64,43 @@ function islandora_compound_object_manage_form($form, &$form_state, $object) {
       ) + islandora_compound_object_get_tabledrag_element($object);
     }
 
-  }
-
-  // Add parents.
-  $form['add_to_parent'] = array(
-    '#type' => 'fieldset',
-    '#title' => t('Associate with Parent Object'),
-    '#description' => t('Add this object to a parent object'),
-  );
-  $form['add_to_parent']['parent'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Parent Object Pid/Label'),
-    '#autocomplete_path' => 'islandora_compound_object/autocomplete/parent',
-  );
-
-  // Remove parents.
-  $parent_part_of = $object->relationships->get('info:fedora/fedora-system:def/relations-external#', $rels_predicate);
-  if (!empty($parent_part_of)) {
-    $form['parents'] = array(
+    // Add parents.
+    $form['add_to_parent'] = array(
       '#type' => 'fieldset',
-      '#title' => t('Unlink From Parent'),
-      '#description' => t('Remove the relationship between this object and parent objects.'),
+      '#title' => t('Associate with Parent Object'),
+      '#description' => t('Add this object to a parent object'),
+    );
+    $form['add_to_parent']['parent'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Parent Object Pid/Label'),
+      '#autocomplete_path' => 'islandora_compound_object/autocomplete/parent',
     );
 
-    $parents = array();
-    foreach ($parent_part_of as $parent) {
-      // Shouldn't be too much of a hit but would be good to avoid the
-      // islandora_object_loads.
-      $pid = $parent['object']['value'];
-      $parent_object = islandora_object_load($pid);
-      $parents[$pid] = array('title' => $parent_object->label, 'pid' => $pid);
-    }
+    // Remove parents.
+    $parent_part_of = $object->relationships->get('info:fedora/fedora-system:def/relations-external#', $rels_predicate);
```

Comment:
Should you be using the FEDORA_RELS_EXT_URI constant instead of a literal string?